iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -370,6 +370,70 @@ static Operation *getInsertionPoint(Block &block, ArrayRef<Value> usedValues) {
   return insertAfter->getNextNode();
 }
 
+/// Modifies `dispatchOp` to attach operand-result tie information when
+/// possible.
+static void tryToTieOperandsAndResults(
+    IREE::Flow::DispatchWorkgroupsOp dispatchOp) {
+  Block *block = dispatchOp.getBody(0);
+  unsigned numResults = dispatchOp.getNumResults();
+  auto inputs = block->getArguments().drop_back(numResults);
+  auto outputs = block->getArguments().take_back(numResults);
+
+  // Returns the tied operand for the given `resultArg`. Returns nullptr
+  // if error or not found.
+  auto getTiledOperandBlockArgument =
```

Comment:
typo: `getTiedOperandBlockArgument?` (here and everywhere else including PR title)

Reply:
Ah, good catch! Was tying enough tiling so my fingers have their own opinions now. :D